components/radio/group.tsx

```diff
@@ -33,7 +33,7 @@ const RadioGroup = React.forwardRef<HTMLDivElement, RadioGroupProps>((props, ref
       className = '',
       options,
       optionType,
-      buttonStyle,
+      buttonStyle = 'outline' as RadioGroupButtonStyle,
```

Comment:
It is different when `<Radio.Group buttonStyle={undefined} />`

Reply:
if it is  undefined [not as string] it still will pick up value from defaultProps,

demo: https://jsfiddle.net/zhnL4xbm/